core/capabilities_test.go

```diff
@@ -14,6 +14,56 @@ import (
 	"pgregory.net/rapid"
 )
 
+func TestCapability_Capacities(t *testing.T) {
+	assert := assert.New(t)
+	// create capabilities
+	capabilities := []Capability{128, 2, 3}
+	mandatory := []Capability{4}
+	caps := NewCapabilities(capabilities, mandatory)
+	// convert to net.Capabilities and back
+	caps = CapabilitiesFromNetCapabilities(caps.ToNetCapabilities())
+	// check all capacities present and set to 1
+	assert.Equal(len(caps.capacities), len(capabilities))
+	for _, c := range capabilities {
+		v, exist := caps.capacities[c]
+		assert.True(exist)
+		assert.Equal(1, v)
+	}
+	// test increase capacity
```

Comment:
Could we split this up into multiple tests? i.e converting to/from net.Capabilities, increasing capacity, decreasing capacity separately

Reply:
Yes we could, but I'm not sure if it's necessary, this test is focused on pretty small piece of code.